spark3/src/main/java/org/apache/iceberg/spark/SparkSessionCatalog.java

```diff
@@ -121,6 +121,15 @@ public Table loadTable(Identifier ident) throws NoSuchTableException {
     }
   }
 
+  @Override
+  public void invalidateTable(Identifier ident) {
+    if (icebergCatalog.tableExists(ident)) {
```

Comment:
Do you need to check for Identififer cases whether is null or is it done by TableExists?

Reply:
This should only be invoked by Scala in Spark so we shouldn't have a null ever passed through basically the same as the other Api's in this class like "loadTable"